modules/olm-building-operator-catalog-image.adoc

```diff
@@ -3,10 +3,10 @@
 // * operators/admin/olm-managing-custom-catalogs.adoc
 
 ifdef::openshift-origin[]
-:registry-image: quay.io/openshift/origin-operator-registry:4.7.0
```

Comment:
This whole module is actually being removed for 4.8 per https://github.com/openshift/openshift-docs/pull/33720, so the changes in the PR here for `modules/olm-building-operator-catalog-image.adoc` can be dropped.